test/it/user-update.js

```diff
@@ -36,11 +36,12 @@ describe('user.update()', () => {
     _user.profile.nickName = 'rob';
     const modified = JSON.parse(JSON.stringify(_user));
     delete modified.lastUpdated;
+    setTimeout(function () {}, 1000);
```

Comment:
I have no idea what the best way to _sleep_ is.  The internets failed me, and returned about 50 different ways, so looking for feedback